src/main/scala/fi/oph/koski/raportointikanta/RaportointiDatabaseSchema.scala

```diff
@@ -44,6 +44,17 @@ object RaportointiDatabaseSchema {
     sqlu"DROP TABLE IF EXISTS raportointikanta_status"
   )
 
+  val grantPermissions = DBIO.seq(
+    sqlu"GRANT SELECT ON TABLE r_opiskeluoikeus TO r_koski_katselija, r_koski_henkilo_katselija",
+    sqlu"GRANT SELECT ON TABLE r_opiskeluoikeus_aikajakso TO r_koski_katselija, r_koski_henkilo_katselija",
+    sqlu"GRANT SELECT ON TABLE r_paatason_suoritus TO r_koski_katselijam r_koski_henkilo_katselija",
+    sqlu"GRANT SELECT ON TABLE r_osasuoritus TO r_koski_katselija, r_koski_henkilo_katselija",
+    sqlu"GRANT SELECT ON TABLE r_organisaatio TO r_koski_katselija, r_koski_henkilo_katselija",
+    sqlu"GRANT SELECT ON TABLE r_koodisto_koodi TO r_koski_katselija, r_koski_henkilo_katselija",
+    sqlu"GRANT SELECT ON TABLE r_raportointikanta_status TO r_koski_katselija, r_koski_henkilo_katselija",
+    sqlu"GRANT SELECT ON TABLE r_henkilo TO r_koski_henkilo_katselija"
```

Comment:
Nämä vois tehdä yhdelläkin rivillä, tyyliin

    grant select on table
    r_opiskeluoikeus, r_opiskeluoikeus_aikajakso, r_paatason_suoritus, ...
    to r_koski_katselija, r_koski_henkilo_katselija